components/Cards/src/ColorThemer/MDCCardsColorThemer.m

```diff
@@ -0,0 +1,31 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCCardsColorThemer.h"
+
+@implementation MDCCardsColorThemer
+
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+                          toCard:(nonnull MDCCard *)card {
+  card.backgroundColor = colorScheme.surfaceColor;
+}
+
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
+                      toCardCell:(nonnull MDCCardCollectionCell *)cardCell {
+  cardCell.backgroundColor = colorScheme.surfaceColor;
```

Comment:
@ianegordon I would love some clarification as I don't fully understand the reasoning. Apple set up the `backgroundView` api to be able to set up a custom view to be placed behind all other subviews. This view needs to be initialized by the user and is null by default in a `UICollectionViewCell`. This means we will need to allocate a new `UIView` subview just to set the cell's `backgroundColor`. I see a few problems with this:

1. An overkill only to have the background color of the cell be themed; Apple has provided us `backgroundColor` for a reason.
2. If users in their code set up a custom `backgroundView` for their cell it will override our view and therefore there will no longer be a themed background color for the cell.
3. backgroundColor works as expected with shapes, as the shapes implementation calls the CAShapeLayer sublayer's colorFill. Whereas backgroundView will set the color on the bounds.

Reply:
Hm. Agreed that we may just want to set the cell's background color then. It gets tricky when there is a backgroundView, but presumably if a client is using a background view then they are aware that it will cover the cell's background.

Reply:
tl;dr: Don't directly use UICollectionCell.view or it's properties to handle state and content.

Apple explicitly extended the API for Table and Collection Cells.
The views that developers should be using for cells are:
contentView
selectedBackgroundView
backgroundView

Otherwise, state changes will not get properly reflected.
See:
https://developer.apple.com/documentation/uikit/uicollectionviewcell?language=objc